_docs/operations/upgrades.md

```diff
@@ -0,0 +1,219 @@
+---
+layout: guide
+title: StorageOS Docs - Upgrade StorageOS version
+anchor: operations
+module: operations/upgrades
+---
+
+# Upgrade StorageOS version
+
+> Before upgrading, check the 
+> [release notes]({%link _docs/reference/release_notes.md %}) to confirm
+> whether there is a safe upgrade path between versions.
+
+
+StorageOS version upgrades must be planned and executed taking into consideration
+that data will be inaccessible during the process. It is recommended to
+schedule a maintenance window.
+
+Currently there are two strategies to upgrade StorageOS and both maintain data
+consistency during the upgrade process. StorageOS keeps the data in the hosts
+where the StorageOS container is running. When the new version of StorageOS starts,
+the volumes from the previous version are available.
+
+1. Full stop of the cluster
+1. Rolling update
+
+> More upgrade procedures will be released that will automate the main part of
+> the process and fulfil use cases not covered currently.
+
+> The [StorageOS cli]({%link _docs/reference/cli/index.md %}) is required to perform an upgrade.
+
+## Option 1. Full stop of the cluster
+
+This option consists of downscaling all applications using StorageOS volumes to
+0 (only those mounting volumes), stop all StorageOS pods, start StorageOS with
+new version and rescale applications to previous size. Deployments that don't
+use StorageOS volumes remain unaffected.
+
+This option will not require moving data across nodes, therefore it is
+recommended for clusters with large data sets. However, __it implies downtime__ for
+stateful applications.
+
+Execute the following instructions:
+
+Prepare upgrade:
+
```

Comment:
We recommend an etcd backup for the second option - why not this one?

Reply:
because if the etcd is deployed as a pod you might run into eviction rainstorm. But in the first option etcd nodes are untouched. I'll add the backup in the first one anyway. No harm